services/image-imgscalr/src/main/java/com/gentics/mesh/image/ImgscalrImageManipulator.java

```diff
@@ -434,6 +433,14 @@ public Completable handleS3CacheResize(String bucketName, String cacheBucketName
 				});
 	}
 
+	private String getS3CachePath() {
```

Comment:
S3 cache violates the overall cache placement rules, so is moved to its own folder.